### ISSUE TITLE
Add more metrics like MAPE and MCC

### ISSUE THREAD

**Issue by aaishwarymishra** [pytorch/ignite#3536]
## 🚀 Feature

I was checking the metrics and found some metrics like Mean Absolute Percentage Error (MAPE) in regression and  Matthews Correlation Coefficient(MCC) for classification missing adding them will be nice

references: 
[MAPE](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_percentage_error.html)
[MCC](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.matthews_corrcoef.html)

**Comment by vfdev-5** [comment]
@aaishwarymishra thanks for the suggestion! 
We can use sklearn metrics with wrapper: https://docs.pytorch.org/ignite/generated/ignite.metrics.EpochMetric.html#ignite.metrics.EpochMetric similar to https://docs.pytorch.org/ignite/_modules/ignite/metrics/average_precision.html#AveragePrecision

If you would like to implement this metric in the "online" way (without having to store the entire model predictions `y_pred` and targets `y_true`), you can give it a try.

**Comment by aaishwarymishra** [comment]
Yeah I can. Also during building an decoder model I noticed that I needed to flatten and apply masking manually, I think adding a accuracy metric wrapper for nlp will be good that does this without needing a extra function.
